src/classes/HiFiCommunicator.ts

```diff
@@ -309,6 +333,9 @@ export class HiFiCommunicator {
         onMuteChanged?: OnMuteChangedCallback,
         connectionRetryAndTimeoutConfig?: ConnectionRetryAndTimeoutConfig
     } = {}) {
+        
+        this._visitId = RaviUtils.createUUID();
```

Comment:
visitid has to be the same across multiple connections, so we cannot use the ravi session id